reconcile/external_resources/factories.py

```diff
@@ -28,6 +29,14 @@
 
 T = TypeVar("T")
 
+AWS_DEFAULT_TAGS = [
+    {
+        "tags": {
+            "app": "app-sre-infra",
```

Comment:
a similar one was added in https://github.com/app-sre/qontract-reconcile/pull/3836 for tracking cost, should we make this a config instead of hard coded `app-sre-infra`?

Reply:
If we need this to be configurable we can add the setting afterwards if needed. 
.- I expect to have all resources tagged with the new modules. 
.- This is just to avoid changes in the migration process to make it smooth.

Reply:
since now it's the second time we use this default value, better to make it configurable later